static/espeasy_default.css

```diff
@@ -1,6 +1,70 @@
-h1,
-h6 {
-	color: #07d
+:root {
+	--bg1: #eee;
+	--bg2: #cacaca;
+	--bg3: #07d;
+	--c1: #fff;
+	--bdc1: silver;
+	--bg4: #e8e8e8;
+	--bg5: #dbff0075;
+	--c2: #444;
+	--bg6: #fff;
+	--bdc2: #444 #444 #fff;
+	--c3: #000;
+	--bg7: #bdbdbd;
+	--bg8: #029b34;
+	--bg9: inherit;
+	--c4: inherit;
+	--c5: #07d;
+	--bg10: #07d;
+	--bg11: #f8f8f8;
+	--c6: #fff;
+}
+
+[data-theme="dark"] {
+	--bg1: #444;
+	--bg2: #2e2e2e;
+	--bg3: #00000085;
+	--c1: #dfdfdf;
+	--bdc1: #545454;
+	--bg4: #2a2c2e;
+	--bg5: #553044;
+	--c2: #fff;
+	--bg6: #222;
+	--bdc2: transparent;
+	--c3: #fff;
+	--bg7: #00000030;
+	--bg8: #30553d;
+	--bg9: #222;
+	--c4: #c3c3c3;
+	--c5: #c3c3c3;
+	--bg10: #44607a;
+	--bg11: #44607a;
+	--c6: #c3c3c3;
+}
+
+@media (prefers-color-scheme:dark) {
+	[data-theme="auto"] {
```

Comment:
Can't you "inherit" from the dark theme here?
Maybe the order should be swapped then to handle the `@media` calls first?

Reply:
Inheritance in CSS is only supported on variable level, not on a group of variables, it's not (really) a programming language, nor is it trying to be, that's what sass / scss are for. (and no, you don't want that, it will generate similar css code, possibly even more bloated) 😃